### PR TITLE
Dev tddominik

### PR DESCRIPTION
**Pull request by domlen2003**
Auto Registration removed but still useful features added

### CODE REVIEW

File: src/main/java/de/th3ph4nt0m/tdbot/interfaces/ICommand.java [hunk 1]
```diff
@@ -22,13 +22,16 @@
...
 
+import de.th3ph4nt0m.tdbot.core.CommandHandler;
 import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
 
 public
 interface ICommand
 {
-    boolean called(String[] args, MessageReceivedEvent event);
-
+    //returns true when command is unsafe to run in current conditions
```
Comment (Th3Ph4nt0m) [inline src/main/java/de/th3ph4nt0m/tdbot/interfaces/ICommand.java:31]: Please use [Javadocs Comments](https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html)

File: src/main/java/de/th3ph4nt0m/tdbot/interfaces/ICommand.java [hunk 2]
```diff
@@ -22,13 +22,16 @@
...
+    boolean unsafe(String[] args, MessageReceivedEvent event);
+    //calls the command
```
Comment (Th3Ph4nt0m) [inline src/main/java/de/th3ph4nt0m/tdbot/interfaces/ICommand.java:33]: Please use [Javadocs comments](https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html)

File: src/main/java/de/th3ph4nt0m/tdbot/core/CommandHandler.java [hunk 3]
```diff
@@ -20,27 +20,108 @@
...
 {
 
-    public static HashMap <String, ICommand> commands = new HashMap <>();
+	public CommandHandler() {
+		addCommand(new CMD_flipcoin());
+		addCommand(new CMD_help());
+		addCommand(new CMD_repo());
+		addCommand(new CMD_userinfo());
+		addCommand(new CMD_version());
+		//addCommands();
```
Comment (Th3Ph4nt0m) [inline src/main/java/de/th3ph4nt0m/tdbot/core/CommandHandler.java:47]: Remove unused methods please.

**Review by Th3Ph4nt0m — CHANGES_REQUESTED** [review]
Please change the points i wrote as comments.

**Comment by Th3Ph4nt0m** [comment]
As you can see, your latest changes to the level system have just been incorporated into this PR, so I cannot accept them. Let's work best with feature branches (e.g. feature/levelsystem) to improve the workflow.

**Comment by domlen2003** [comment]
Könnten der liebe Herr Ph4nt0m bitte den verkackten pullrequest annehmen.

**Comment by domlen2003** [comment]
Wäre coll thx

**Review by Th3Ph4nt0m — APPROVED** [review]
Tested without a database, but should work.

**Pull request merged**